Code/Framework/AzCore/AzCore/Math/Internal/SimdMathVec1_scalar.inl

```diff
@@ -434,9 +434,28 @@ namespace AZ
             return atan2f(y, x);
         }
 
+
         AZ_MATH_INLINE Vec1::FloatType Vec1::ExpEstimate(FloatArgType x)
         {
-            return exp(x);
+            static constexpr float   ExpCoef1 = 1.2102203e7f;
+            static constexpr int32_t ExpCoef2 = -8388608;
+            static constexpr float   ExpCoef3 = 1.1920929e-7f;
+            static constexpr float   ExpCoef4 = 3.371894346e-1f;
+            static constexpr float   ExpCoef5 = 6.57636276e-1f;
+            static constexpr float   ExpCoef6 = 1.00172476f;
+
+            const int32_t a = int32_t(ExpCoef1 * x);
+            const int32_t b = a & ExpCoef2;
+            const int32_t c = a - b;
+            const float f = ExpCoef3 * float(c);
+            const float i = f * ExpCoef4 + ExpCoef5;
+            const float j = i * f + ExpCoef6;
+            uint32_t k;
+            memcpy(&k, &j, sizeof(float));
+            k += b;
+            float r;
+            memcpy(&r, &k, sizeof(float));
+            return r;
```

Comment:
Why not just use a union or a reinterpret_cast<> here? 
```
return reinterpret_cast<float>(reinterpret_cast<uint32_t>(j) + b);
```

Reply:
To avoid undefined behaviour across various compilers and platforms. On many architectures a union or reinterpret will work fine, but to guarantee correctness a memcpy is required. Although, I see std::bit_cast<> was added in 20, that could possibly work here as well..

Reply:
Reinterpret cast would work if used for a `float&`, it will not work for a `float` as it wouldn't even compile.
`return reinterpret_cast<float&>(reinterpret_cast<uint32_t&>(j) + b);`